spec/view/post_index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'test post/index', type: :feature do
+  before(:all) do
+    @user = User.find(2)
+    @posts = Post.where(author_id: @user.id).includes(comment: [:author])
+  end
+  scenario 'posts photo test' do
+    visit user_post_index_path(2)
+    expect(page).to have_xpath("//img[@src='#{@user.photo}']")
+  end
+  scenario 'users name test' do
```

Comment:
As suggested by the previous reviewer, most of your tests are still failing. Please check the image below :point_down: and see the ones failing and kindly work on them. 

![Screenshot from 2023-03-21 10-56-34](https://user-images.githubusercontent.com/65233860/226587451-ff8e348f-374d-4c5e-a31d-437c25445c74.png)


![Screenshot from 2023-03-21 10-56-08](https://user-images.githubusercontent.com/65233860/226587419-6d3a3dda-f815-44e0-9e49-765cbbfd53e2.png)